contributing/content-style.md

```diff
@@ -399,6 +401,12 @@ something that makes sense out of the context of the rest of the page.
 Remember to keep it short.
 If possible, it should be no more than about 160 characters.
 
+## Use shortcodes for versions and dates
+
+When referencing service or runtime versions in documentation, always use the shortcode {{% latest "xxx" %}} instead of hardcoding version numbers. 
```

Comment:
```suggestion
When referencing service or runtime versions in documentation, always use the shortcode `{{% latest "xxx" %}}` instead of hardcoding version numbers. 
```